scilifelab_epps/wrapper.py

```diff
@@ -48,7 +48,10 @@ def epp_wrapper(args):
             process = Process(lims, id=args.pid)
 
             # Get EPP user
-            epp_user: Researcher = get_epp_user(lims, args.pid)
+            try:
+                epp_user: Researcher = get_epp_user(lims, args.pid)
+            except:
```

Comment:
What exception is it throwing?

Reply:
1194c0f53bfd891f7e5a37c6edf270a1da52d152